ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -90,41 +90,40 @@
 
 ### BGP Neighbors
 
-| Neighbor | Remote AS | VRF |
-| -------- | --------- | --- |
+| Neighbor | Remote AS | VRF | Send-community | Maximum-routes |
+| -------- | --------- | --- | -------------- | -------------- |
 {%         for neighbor in router_bgp.neighbors | arista.avd.natural_sort %}
-{%             if router_bgp.neighbors[neighbor].remote_as is arista.avd.defined %}
-| {{ neighbor }} | {{ router_bgp.neighbors[neighbor].remote_as }} | default |
-{%             elif router_bgp.neighbors[neighbor].peer_group is arista.avd.defined %}
-| {{ neighbor }} | Inherited from peer group {{ router_bgp.neighbors[neighbor].peer_group }} | default |
+{%             if router_bgp.neighbors[neighbor].peer_group is arista.avd.defined %}
```

Comment:
Logic here will prefer peer group values instead of neighbor values.
I suggest to keep the `inherited.xxx` values as you have, and then utilize the `arista.avd.default` filter's ability to take multiple default values. So `{{ router_bgp_neighbors[neighbor]].remote_as | arista.avd.default(inherited.remote_as, '-') }}`
To avoid the output line from being too long, you could also set a `row.remote_as` variable. It will make it easier to read and maintain.